### PR TITLE
Add missing Address-of-Operator

### PR DESCRIPTION
**Pull request by aeharvlee**
## What type of PR is this?

_(REQUIRED)_

- [ x ] bug
- [ ] cleanup  
- [ ] documentation
- [ ] feature

## What this PR does / why we need it:
Without Address-of-Operator, it dose not work.

## Which issue(s) this PR fixes:

_(REQUIRED)_

## Special notes for your reviewer:

_(fill-in or delete this section)_

## Testing

_(fill-in or delete this section)_

## Release Notes

_(REQUIRED)_

```release-note

```

### CODE REVIEW

File: docs/v1/manual.md [hunk 1]
```diff
@@ -221,7 +221,7 @@ func main() {
   app := cli.NewApp()
 
   app.Flags = []cli.Flag {
-    cli.StringFlag{
+    &cli.StringFlag{
```
Comment (rliebz) [inline docs/v1/manual.md:224]: This is a difference between the `v1` and `v2` versions of the API. These are the docs for `v1`, in which the non-pointer version of the `cli` flags implement the interface.

Comment (aeharvlee) [reply docs/v1/manual.md:224]: I met this bug with cli v1.22.4 with go 1.14.
I can show you all of source code, go.mod everything I used.

Where should I submit a full bug template to this PR or issues?

Thanks for your feedback.

Comment (rliebz) [reply docs/v1/manual.md:224]: GitHub links are weird, but I think this is it: https://github.com/urfave/cli/issues/new?assignees=&labels=kind%2Fbug%2C+status%2Ftriage%2C+area%2Fv1&template=v1-bug-report.md&title=v1+bug%3A+%28+your+bug+title+goes+here+%29

If that doesn't work, on this repo, click "New Issue" > "v1 bug report", and the template should be filled out for you

Comment (aeharvlee) [reply docs/v1/manual.md:224]: I missed it. You're right. Without Address-of-Operator it does work. 
go run main.go / go build main.go both is OK, but my IDE GoLand still show me syntax error.

Anyway Thanks for your feedback. 
It does work. Thanks.

**Pull request closed without merging**